src/main/java/org/jabref/logic/importer/fetcher/GrobidCitationFetcher.java

```diff
@@ -64,9 +64,8 @@ private String parseUsingGrobid(String plainText) throws FetcherException {
               .map(String::trim)
               .collect(Collectors.toCollection(TreeSet::new));
         if (plainReferences.size() == 0) {
-            throw new FetcherException("Your entered references are empty.");
+            return List.of();
```

Comment:
`Collections.emptyList();`

#evenmorenitpicl

Reply:
Please do not use `.size() == 0`, `.size() > 0`. Please use `.isEmpty()`. Please check the whole code using Strg+Shift+F in IntelliJ.

Reply:
This was already fixed here: deb65c000c8f6b1ff1b362d265f5fedfec721671 and will soon be merged into this branch.
We will check the whole code when no more conversations are left.